posts/2021-the-bottom-line.md

```diff
@@ -0,0 +1,177 @@
+---
+author: 'Mathlib community'
+category: 'overview'
+date: 2021-12-30 23:59:59+00:00
+description: ''
+has_math: true
+link: ''
+slug: 2021-the-bottom-line
+tags: ''
+title: 2021 - The Bottom Line
+type: text
+---
+
+Since the mathlib repository was created in summer 2017,
+each year has been bigger than the last. 
+As an end-of-year retrospective, we look at how the mathlib library and community have developed in 2021.
+
+# Statistics
+
+Since early 2019, mathlib has followed a "squash and merge" strategy for pull requests.
+One commit to the main branch corresponds to one merged pull request.
+
+| | Dec 31, 2020 | Dec 31, 2021 | Increase |
+| | -------------- | ------------ | |
+| Number of files | 1498 | 2329 | 831 (55%) |
+| Lines of code in repository [^1] | 455152 | 745259 | 290107 (64%) |
+| SLOC in `/src` [^1] | 284441 | 473644 | 189203 (67%) |
+| Lines of comments | 65219 | 108658 | 43439 (67%) |
+| Total commits | 5751 | 10960 | 5209 (91%) |
+| Definitions | 15437 | 24861 | 9424 (61%) |
+| Theorems | 44038 | 72847 | 28809 (64%) | 
+| Contributors | 132 | 209 | 77 (58%) |
+
+[^1]: We count source lines of code using [`cloc`](http://cloc.sourceforge.net/). This count includes only non-comment, non-whitespace lines of Lean code in the `/src` directory of mathlib. The "lines of code in repository" count includes blank and comment lines in all file types in all directories of the mathlib repository.
+
+For more information, see the page on [mathlib statistics](https://leanprover-community.github.io/mathlib_stats.html).
+
+## Records
+
+
+December 2021 set a record as the month with the highest number of commits, at 565. 
+Four of the five most active months happened in 2021;
+December, October, and September were mathlib's first, second, and third most active months, respectively.
+
+Yury Kurdryashov was the year's most prolific contributor, with 662 commits, surpassing his previous year's record of 624.
```

Comment:
```suggestion
Yury Kudryashov was the year's most prolific contributor, with 662 commits, surpassing his previous year's record of 624.
```

Reply:
Oops, sorry Yury! A typo, I know how to spell your name :)